File: 20220831/모범답안/20220831_02.js/CountContinuousBinary.js
/**
 * @param arr {number[]}
 * @returns {number}
 */
function solution(arr) {
  let result = 0
  let continuedOneCount = 0
  for (const binary of arr) {
    if (binary === 1) {
      continuedOneCount++
      result = Math.max(result, continuedOneCount)
    } else {
      continuedOneCount = 0
    }
  }
  return result
}

solution

File: 20220831/모범답안/20220831_02.js/CountContinuousBinary.py
def solution(arr):
    """
    :param arr: int[]
    :return: int
    """

    result = 0
    continuedOneCount = 0
    for binary in arr:
        if (binary == 1):
            continuedOneCount += 1
            result = max(result, continuedOneCount)
        else:
            continuedOneCount = 0
    return result
